libs/hwes/event.go

```diff
@@ -18,6 +21,11 @@ type Event struct {
 	Data          []byte
 	Timestamp     time.Time
 	Version       uint64
+	UserID        *uuid.UUID
+}
+
+type metadata struct {
+	UserID string `json:"user_id"`
```

Comment:
Who's user id? The name does not reflect, that we mean the "committer" or something

Reply:
https://github.com/helpwave/services/issues/682
> We should store the users identity via their ID in the our EventStore as well to track the actual caller.

Thats then more of a general discussion. The goal is to track the identity that is directly responsible for this event.
- `UserID` assumes that an event can be triggered by a user. Thats true (for now).
- `CommitterID` would move the whole user/identity concept out of `hwes`. 
- `IdentityID` would bind more loosely these two concepts, an event and a user/identity. But then, its not clear, right now, if the `IdentityID` represents a user or for example some sort of service account.

I recommend: For the metadata, `CommiterID` should be good but gets then translated to `UserID` in `hwes.Event`.